### ISSUE TITLE
Windows: native optional dependencies that are marked excluded still fails the installation

### ISSUE THREAD

**Issue by kenotron** [yarnpkg/yarn#1520]
Even after #525 has been closed, I still see this bug where it fails to install node_modules because of native optional deps like fsevent on Windows

```
warning fsevents@1.0.14: The platform "win32" is incompatible with this module.
info "fsevents@1.0.14" is an optional dependency and failed compatibility check. Excluding it from installation.
error Error running install script for optional dependency: "E:\\path\\to\\node_modules\\fsevents: Command failed.
```

**Comment by TrySound** [comment]
Ref https://github.com/yarnpkg/yarn/issues/1285

**Comment by Daniel15** [comment]
This looks like a duplicate of #1285, let's track the issue over there instead. Thank you for trying Yarn! 😄
